### PR TITLE
Remove all *.trx files before executing

### LINKED ISSUE #6488: dotnet test publish results - duplicates previous tasks' results
1. Create two test projects
2. Create build task ```dotnet test proj1.tests.csproj``` with publish enabled
3. Create build task ```dotnet test proj2.tests.csproj``` with publish enabled
4. Run build
5. Results from `proj1.tests.csproj` are published twice

Log for step 2 looks like this:
```
2018-02-21T14:51:44.4918390Z Results File: D:\a\_temp\VssAdministrator_factoryvm-az265_2018-02-21_14_51_43.trx
2018-02-21T14:51:44.9541003Z ##[section]Async Command Start: Publish test results
2018-02-21T14:51:45.0399232Z Publishing test results to test run '258'
2018-02-21T14:51:45.0399456Z Test results remaining: 202. Test run id: 258
2018-02-21T14:51:45.6759084Z Published Test Run : xxxxxxxxxxxxxx
2018-02-21T14:51:45.6759464Z ##[section]Async Command End: Publish test results
```

Log for step 3 looks like this:
```
2018-02-21T14:52:03.4059924Z Results File: D:\a\_temp\VssAdministrator_factoryvm-az265_2018-02-21_14_52_02.trx
2018-02-21T14:52:03.5848043Z ##[section]Async Command Start: Publish test results
2018-02-21T14:52:03.6071384Z Publishing test results to test run '259'
2018-02-21T14:52:03.6071535Z Test results remaining: 202. Test run id: 259
2018-02-21T14:52:03.9313838Z Published Test Run : xxxxxxxxxxxxxx
2018-02-21T14:52:08.1417901Z Publishing test results to test run '260'
2018-02-21T14:52:08.1418097Z Test results remaining: 83. Test run id: 260
2018-02-21T14:52:08.5283796Z Published Test Run : xxxxxxxxxxxxxx
2018-02-21T14:52:08.5284410Z ##[section]Async Command End: Publish test results
```

Seems that when the task is flagged to publish the results, it publishes `D:\a\_temp\*.trx` instead of just the trx's from the task? Just a guess.

### PR DESCRIPTION
**Pull request by Mardoxx**
Fixes #6488 

Please advise on how to test this and I will.

### CODE REVIEW

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 1]
```diff
@@ -101,6 +101,7 @@ export class dotNetExe {
         const resultsDirectory = tl.getVariable('Agent.TempDirectory');
 
         if (enablePublishTestResults && enablePublishTestResults === true) {
+            tl.rmRF(`${resultsDirectory}/*.trx`);
```
Comment (nigurr) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: rmRF doesn't support this. We need to find the files and if they exist, pass it to rmRF function. 
Please check for doc: https://github.com/Microsoft/vsts-task-lib/blob/master/node/task.ts
https://github.com/Mardoxx/vsts-tasks/blob/8b361827f14aed92de1cd0cc05df3ab136aadbcb/docs/contribute.md

**Comment by Mardoxx** [comment]
Cool thanks

**Comment by stephenmichaelf** [comment]
@nigurr Can this be closed?

**Comment by Mardoxx** [comment]
Fized in #6615

**Pull request closed without merging**